core/src/main/java/greencity/service/impl/UserServiceImpl.java

```diff
@@ -621,6 +621,9 @@ public User updateUserProfilePicture(MultipartFile image, String email) {
         User user = userRepo
             .findByEmail(email)
             .orElseThrow(() -> new WrongEmailException(USER_NOT_FOUND_BY_EMAIL + email));
+        if (image == null) {
```

Comment:
How the image can be null here if you added ImageValidation at controller level? Is ImageValidation working?